content/doc/developer/handling-requests/_chapter.yml

```
@@ -3,3 +3,4 @@ sections:
 - routing
 - actions
 - responses
+- deprecated-reflective-access
```

```suggestion
- stapler-deprecated-reflective-access
```

Plus corresponding changes below, in case we could need more than just Stapler, or just to disambiguate.